crowdsec-docs/unversioned/console/remediation_metrics.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: remediation_metrics
+title: Remediation Metrics
+---
+
+:::info
+To receive remediation metrics, make sure you’re using **CrowdSec v1.6.3** or higher.  
+**Note:** Not all Remediation Components report metrics to the CrowdSec Console. For details, refer to the [official documentation](https://docs.crowdsec.net/u/bouncers/intro).
+:::
+
+## Introduction
+
+The **Remediation Metrics** page offers a clear and comprehensive view of the malicious activity that CrowdSec has detected and blocked on your infrastructure.  
+It provides key insights into:
+- The number and types of attacks
+- The impact of remediation measures
+- The estimated resources saved by stopping these threats
+
+The page is divided into three main sections:
+- **Malicious Intents** – A breakdown of attack types associated with IPs blocked by security engines and blocklists.
+- **Malicious Traffic Discarded** – Raw and estimated data showing how much malicious traffic has been dropped by your remediation components.
+- **Projected Resources Saved** – An estimate of the resources preserved thanks to traffic being blocked (e.g., storage, bandwidth, log volume).
+
+---
+
+## Malicious Intents
+
+At the top of the page, you'll see the **Total Prevented Attacks** for the selected time period. This gives you an immediate overview of how many threats CrowdSec has detected and blocked.
```

Comment:
Not all remediations "block" we need to switch most wording to say "detected and remediated" or even "dropped" as that is primary key word on the page.